### PR TITLE
make convert.py has same procedure with run_pretrained_models.py

### PR DESCRIPTION
**Pull request by zhijxu-MS**

### CODE REVIEW

File: tf2onnx/convert.py [hunk 1]
```diff
@@ -145,10 +145,10 @@ def main():
                              output_names=outputs,
                              inputs_as_nchw=args.inputs_as_nchw)
 
-    model_proto = g.make_model("converted from {}".format(model_path))
-
-    logger.info("")
-    model_proto = GraphUtil.optimize_model_proto(model_proto)
+    onnx_graph = optimizer.optimize_graph(g)
+    if not onnx_graph:
```
Comment (nbcsm) [inline tf2onnx/convert.py]: optimizer.optimize_graph should always return a valid graph, no need to check None here.

you may want to add a None check at https://github.com/onnx/tensorflow-onnx/blob/d239d1cc5f73d45be93b2f2160d408253538871d/tf2onnx/optimizer/__init__.py#L50
something like ```graph = opt.optimize(current) or graph```

and we can skip None check here.

Comment (zhijxu-MS) [reply tf2onnx/convert.py]: done.

File: tf2onnx/convert.py [hunk 2]
```diff
@@ -14,7 +14,7 @@
 
 import tensorflow as tf
 
-from tf2onnx.graph import GraphUtil
+from tf2onnx import optimizer
```
Comment (nbcsm) [inline tf2onnx/convert.py]: merge with ```from tf2onnx import constants, loader, logging, utils```?

Comment (zhijxu-MS) [reply tf2onnx/convert.py]: done

**Pull request merged**